app/javascript/src/components/editor/Settings.svelte

```diff
@@ -81,6 +81,15 @@
         </div>
       </div>
 
+      <div class="checkbox tooltip mt-1/8">
+        <input id="autocomplete-semicolon" type="checkbox" bind:checked={$settings["autocomplete-semicolon"]} />
```

Comment:
This setting also needs to be added to the defaults further up the file, it should default to true in my opinion

Reply:
It was, wasn't it? https://github.com/Mitcheljager/workshop.codes/pull/419/commits/6edec229360f7de2d6799234d2800618330cc156#diff-a3c60ef0735190d4f10ce80e4f0fa56665aebc6d548cd541449b0e3e4bf4c347R108

Reply:
Ah yes, my bad